packages/dashboard/src/components/journeys/edgeTypes/workflowEdge.tsx

```diff
@@ -47,11 +52,55 @@ export default function WorkflowEdge({
   const nodes = useAppStore((state) => state.journeyNodes);
   const nodesIndex = useAppStore((state) => state.journeyNodesIndex);
   const targetNode = findNode(target, nodes, nodesIndex);
+
+  let sourceWithoutChildSuffix = source;
+  sourceWithoutChildSuffix = sourceWithoutChildSuffix.replaceAll(
+    "-child-0",
+    "",
+  );
+  sourceWithoutChildSuffix = sourceWithoutChildSuffix.replaceAll(
+    "-child-1",
+    "",
+  );
+
+  const journeyId = typeof path.query.id === "string" ? path.query.id : "";
+  const stats = journeyStats[journeyId]?.nodeStats[sourceWithoutChildSuffix];
+
   if (targetNode && targetNode.type === "empty") {
     markerEnd = undefined;
   }
 
   const isPlusVisible = isDragging && !isDropzoneActive;
+
+  const getLabelText = (): string => {
+    if (stats?.type === NodeStatsType.SegmentSplitNodeStats) {
+      if (id.includes("child-0")) {
+        return (100 - stats.proportions.falseChildEdge).toString();
+      }
+      if (id.includes("child-1")) {
+        return stats.proportions.falseChildEdge.toString();
```

Comment:
Can you round the percentages here to 1 decimal point of precision e.g. 5.1 % so we don't get very long percentages as portrayed at the beginning of this video?

https://github.com/dittofeed/dittofeed/pull/612#issuecomment-1925048064